templates/web/base/report/_item.html

```diff
@@ -46,7 +46,7 @@ <h3 class="item-list__heading">[% problem.title | html %]</h3>
             [% END %]
             [%- IF c.cobrand.moniker != 'fixamingata' %] [%# Default: %]
                 [%- IF problem.days_ago <= 28 %]
-                  [% tprintf(loc('Reported %s %s ago'), problem.days_ago, nget( loc('week'), loc('weeks'), problem.days_ago ) ) %]
+                  [% tprintf( nget( loc('Reported %s week ago'), loc('Reported %s weeks ago'), problem.days_ago), problem.days_ago ) %]
```

Comment:
The variable is called days_ago, but the string implies it will be outputting weeks? Should it be 'days' in the string?
Also, if it's a number, should it be %d in the format string? Don't suppose it matters.
Sorry, haven't had/got time at the moment to look at the rest.